winpr/libwinpr/synch/wait.c

```diff
@@ -109,7 +109,7 @@ static long long ts_difftime(const struct timespec *o,
 #if !defined(HAVE_PTHREAD_GNU_EXT)
 #include <pthread.h>
 
-#if defined(__FreeBSD__) || defined(sun)
+#if defined(__FreeBSD__) || defined(sun) || defined(__OpenBSD__)
```

Comment:
OK.  I modified the code and re-tested.  The change has been pushed to my
branch in my fork of the repo.

Does everything look good?


Thanks,
Bryan

On Tue, Apr 21, 2015 at 4:03 PM, akallabeth <notifications@github.com>
wrote:

> In winpr/libwinpr/synch/wait.c
> <https://github.com/FreeRDP/FreeRDP/pull/2562#discussion_r28816377>:
>
> > @@ -109,7 +109,7 @@ static long long ts_difftime(const struct timespec *o,
> >  #if !defined(HAVE_PTHREAD_GNU_EXT)
> >  #include <pthread.h>
> >
> > -#if defined(__FreeBSD__) || defined(sun)
> > +#if defined(__FreeBSD__) || defined(sun) || defined(__OpenBSD__)
>
> @bceverly <https://github.com/bceverly> Just push the commit to this
> branch.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeRDP/FreeRDP/pull/2562/files#r28816377>.
>